src/patient-registration/patient-registration.component.tsx

```diff
@@ -205,6 +196,11 @@ export const PatientRegistration: React.FC = () => {
       }
       return ids;
     }, []);
+
+    let addressObj = {};
```

Comment:
Is this the object that will hold the dynamic address field(s) values? If so, wouldn't it be better to name it `addresses` (that's if the variable name isn't taken in this scope)?

Reply:
Well `addresses` does not seem fully adequate - maybe `addressFields` or `addressFieldValues`?